zh/enable-tls-for-mysql-client.md

```diff
@@ -0,0 +1,464 @@
+---
+title: 为 Kubernetes 上的 TiDB 集群的 MySQL 客户端开启 TLS
+category: how-to
+---
+
+# 为 Kubernetes 上的 TiDB 集群的 MySQL 客户端开启 TLS
+
+本文主要描述了如何为 Kubernetes 上的 TiDB 集群的 MySQL 客户端开启 TLS。
+
+
+- [概述](#概述)	
++ 为 TiDB 集群生成两套证书
+  - [用户自己提供证书](#用户自己提供证书)
+  - [使用 cert-manager 颁发证书](#使用-cert-manager-颁发证书)
+- [创建 TiDB 集群](#创建-TiDB-集群)
+- [MySQL 客户端使用方法](#MySQL-客户端使用方法)
+
+## 概述
+
+TiDB Operator 从 v1.1 开始已经支持为 Kubernetes 上的 TiDB 集群开启 MySQL 客户端 TLS。
+
+开启此功能的步骤为：
+
+- 为即将创建的 TiDB 集群生成两套证书：为 TiDB Server 创建一套 Server
+  端证书，为 MySQL Client 创建一套 Client
+  端证书。有多种生成证书的方式：用户自己提供证书，使用 K8s
+  内置证书颁发系统或者使用 `cert-manager` 颁发证书；
+- 创建两个 K8s Secret 对象，这两个 Secret 对象分别包含上述创建的 Server
+  端证书和客户端证书，这两个 Secret 的名字为
+  `<cluster-name>-tidb-server-secret` 和
+  `<cluster-name>-tidb-client-secret`； 
+- 在创建集群的时候开启此功能。
+
+## 为 TiDB 集群生成两套证书
+
+### 用户自己提供证书
+
+需要创建两套证书：为 TiDB Server 创建一套 Server 端证书，为 MySQL Client
+创建一套 Client 端证书。可以使用官网提供的方式创建证书或者使用 `cfssl`
+创建证书。
+
+#### 官网创建证书
+
+请参考官网文档：[Enable TLS for MySQL Clients | TiDB Documentation](https://pingcap.com/docs/stable/how-to/secure/enable-tls-clients/)。
+
+#### 使用 cfssl 创建证书
+
+##### 下载 cfssl 软件并初始化证书颁发机构
+
+{{< copyable "shell-regular" >}}
+
+``` shell
+mkdir -p ~/bin
+curl -s -L -o ~/bin/cfssl https://pkg.cfssl.org/R1.2/cfssl_linux-amd64
+curl -s -L -o ~/bin/cfssljson https://pkg.cfssl.org/R1.2/cfssljson_linux-amd64
+chmod +x ~/bin/{cfssl,cfssljson}
+export PATH=$PATH:~/bin
+
+mkdir -p ~/cfssl
+cd ~/cfssl
+cfssl print-defaults config > ca-config.json
+cfssl print-defaults csr > ca-csr.json
+```
+
+##### 配置 CA 选项
+
+{{< copyable "shell-regular" >}}
```

Comment:
The code block with `copyable` snippet should be the command that users need to copy and past in operation.
Please confirm that the following code block conforms to this rule.